package.json

```diff
@@ -89,12 +89,12 @@
       "packages/fluentui/*"
     ],
     "nohoist": [
-      "packages/web-components/typescript",
-      "packages/web-components/typescript/**",
-      "packages/web-components/ts-loader",
-      "packages/web-components/ts-loader/**",
+      "packages/web-components/@microsoft/api-extractor",
+      "packages/web-components/@microsoft/api-extractor/**",
+      "packages/web-components/@microsoft/eslint-config-fast-dna",
       "packages/web-components/ts-node",
       "packages/web-components/ts-node/**",
+      "packages/web-components/typescript",
```

Comment:
This is not new, but I moved it down, and removed the `typescript/**` line since TS doesn't have dependencies.